### PR TITLE
Remove azureml-designer-core reference and refine input file type as CSV and Parquet

### PR DESCRIPTION
**Pull request by sccc19**
@guinao @conhua @weedqian @thy09 

This PR makes following changes to anomaly detector custom module：
- remove `azureml-designer-core` reference
- change `DataFrameDirectory `to `AnyDirectory`, change `ColumnPicker `to `String`
- enable input data type to be csv file, parquet file, or path that contains multiple csv files or parquet files
- remove `compute_stats_in_visualization` parameter since it is defined in `azureml-designer-core`
- use the latest version of azureml-sdk and azureml-dataprep to enable module output

### CODE REVIEW

File: aml_module/invoker.py [hunk 1]
```diff
@@ -36,67 +34,92 @@ def is_timestamp_ascending(timestamps):
...
 
 
-def invoke(input_path, detect_mode, timestamp_column, value_column, batch_size, threshold, sensitivity,
-            appendMode, compute_stats_in_visualization, output_path):
-    data_frame_directory = load_data_frame_from_directory(input_path)
+def read_as_dataframe(input_path: str):
+    if os.path.isfile(input_path):
+        if input_path.endswith(".csv"):
+            return pd.read_csv(input_path)
+        elif input_path.endswith(".parquet"):
```
Comment (guinao) [inline aml_module/invoker.py:41]: Why does parquet a data type that we want to support?

File: aml_module/invoker.py [hunk 2]
```diff
@@ -36,67 +34,92 @@ def is_timestamp_ascending(timestamps):
...
+
+        csv_files = list(dir_path.glob("**/*.csv"))
+        if csv_files:
+            df_from_csv_files = (pd.read_csv(f) for f in csv_files)
+            return pd.concat(df_from_csv_files, ignore_index=True)
+
+        parquet_files = list(dir_path.glob("**/*.parquet"))
+        if parquet_files:
+            df_from_parquet_files = (pd.read_parquet(f) for f in parquet_files)
+            return pd.concat(df_from_parquet_files, ignore_index=True)
```
Comment (guinao) [inline aml_module/invoker.py:54]: why do we need these complicated data processing steps?

**Comment by guinao** [comment]
>   namespace: microsoft.com/office

bump up the version to 1.0.1, please also check that the module can run with no error in aml.

---
Refers to: aml_module/module_spec.yaml:4 in 4c55256. [](commit_id = 4c552568db4a3125869a5c8e50f88ff2a36b2d6e, deletion_comment = False)

File: aml_module/invoker.py (hunk 1 above)
Comment (sccc19) [reply aml_module/invoker.py:41]: We just want to support the case that when data type is parquet, the module can process it properly.

File: aml_module/invoker.py (hunk 2 above)
Comment (sccc19) [reply aml_module/invoker.py:54]: And also, more support to the case that multiple files in a folder can be processed. These goals are raised by @weedqian, maybe he can explain more about why we choose such input types.

File: aml_module/invoker.py (hunk 1 above)
Comment (guinao) [reply aml_module/invoker.py:41]: As there are scenarios with this kind of data source, just keep these codes.

File: aml_module/invoker.py (hunk 2 above)
Comment (guinao) [reply aml_module/invoker.py:54]: As there are scenarios with this kind of data source, just keep these codes.

**Pull request merged**